package.json

```diff
@@ -44,25 +45,25 @@
     "@types/twitter": "^1.7.4",
     "@types/ws": "^8.5.3",
     "@wfcd/eslint-config": "^1.4.1",
+    "c8": "^8.0.1",
```

Comment:
Could you explain the reasoning behind moving to c8? I usually dont keep track of these kind of packages.

Reply:
yeah, nyc loses the coverage on esm, c8 doesn't.